agent/engine/dependencygraph/graph.go

```diff
@@ -25,11 +25,13 @@ var log = logger.ForModule("dependencygraph")
 // Because a container may depend on another container being created
 // (volumes-from) or running (links) it makes sense to abstract it out
 // to each container having dependencies on another container being in any
-// perticular state set. For now, these are resolved here and support only
+// particular state set. For now, these are resolved here and support only
 // volume/link (created/run)
 
 // ValidDependencies takes a task and verifies that it is possible to allow all
-// containers within it to reach the desired status by proceeding in some order
+// containers within it to reach the desired status by proceeding in some
+// order.  ValidDependencies is called during DockerTaskEngine.AddTask to
```

Comment:
[You're both right](https://en.wikipedia.org/wiki/Sentence_spacing) :space_invader: